frontend/pages/software/ManageSoftwarePage/ManageSoftwarePage.tsx

```diff
@@ -224,7 +327,7 @@ const ManageSoftwarePage = ({
         buttons={renderHeaderButtons}
       />
     );
-  }, [router, location]);
+  }, [router, location, isLoadingSoftwareVulnerabilitiesWebhook]);
```

Comment:
isLoadingSoftwareVulnerabilitiesWebhook is needed as a dependency because renderHeaderButtons uses isLoadingSoftwareVulnerabilitiesWebhook and was causing inconsistencies with loading state if it wasn't defined here